20_kata.py

```diff
@@ -0,0 +1,15 @@
+# https://www.codewars.com/kata/55c6126177c9441a570000cc
+
+import re
+
+
+def order_weight(strng):
+
+    cleaned_string = re.findall(r"\d+", strng)
```

Comment:
Please change the name of this variable. The variable is a list, but the name indicates that it's a string. Additionally try to come up with the name that will indicate what's stored in it.

Reply:
I think in remarks there was statement that words might be separated by unique whitespace. Okey, name will changed